.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug
+about: File a bug to help us improve ML.NET
+title: ''
+labels: ''
+assignees: ''
```

Comment:
I'm curious what these are here for instead of just using the ones built into github? Or is it so that the submitter can put stuff there since they can't in github?

Reply:
I went through the GitHub UI to create these templates, and there were options to add title, labels, and assignees:

![image](https://user-images.githubusercontent.com/10437687/122807319-43cbab00-d280-11eb-9833-e7ff66f264c3.png)

I just left them blank. Should I delete them from the markdown file too?

Reply:
We can leave them and see what happens. Are you thinking if they add an assignee or label that it will auto add it for us?

Reply:
Yes I think if we set it to something then it auto-adds

Reply:
@michaelgsharp is there something else I need to do to get it approved for merge?